scripts/honcho_oauth_setup.py

```
@@ -0,0 +1,160 @@
+        sys.exit(1)
+    return token
+
+
+def _codex_is_installed() -> bool:
+    return shutil.which("codex") is not None
+
+
+def _codex_login_status() -> bool:
+    """Return True if Codex reports being logged in."""
+    result = subprocess.run(
+        ["codex", "login", "status"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)